src/app/public/modules/file-attachment/file-attachment.component.scss

```diff
@@ -0,0 +1,57 @@
+@import "~@skyux/theme/scss/mixins";
+@import "~@skyux/theme/scss/_compat/mixins";
+
+.sky-file-drop-required {
+  box-shadow: 0 0 8px rgba(239, 64, 68, 0.6);
+  border: 1px solid #ef4044;
```

Comment:
I'm not positive, but can't you use the `$sky-highlight-color-danger` variable here?